plugins/hdf/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#
+# HDF plugin CMake configuration
+#
+
+include (${PDAL_CMAKE_DIR}/hdf5.cmake)
+
+
+if (NOT PDAL_HAVE_HDF5)
+    message(FATAL "HDF5 not found but is required for HDF.")
+else()
```

Comment:
you can remove this else() conditional entirely. You've already kicked them out above this.